mindpype/graph.py

```diff
@@ -133,7 +133,7 @@ def verify(self):
         self._verified = True
 
         # cleanup any data used within verification that are no longer needed
-        self.session.free_unreferenced_data()
+        self.session._free_unreferenced_data()
```

Comment:
it didn't occur to me to include these as public API methods, but methods like this (and `add_graph`) that are not exclusively called from within `Session` methods should also be public.

Reply:
Would add_data be another method where this applies?